### PR TITLE
[DEVHUB-1591] - Point Feedback and Request Content to use devcenter-backend

### PR DESCRIPTION
**Pull request by mcolella14**
## Jira Ticket:

[DEVHUB-1591](https://jira.mongodb.org/browse/DEVHUB-1591)

## Description:

Point to backend service for the POST/PUT requests, along with minor related changes.

Will not merge until this is in production https://github.com/10gen/devcenter-backend/pull/28

### CODE REVIEW

File: src/components/request-content-modal/request-content.modal.tsx
Comment (harikakotipalli) [inline src/components/request-content-modal/request-content.modal.tsx:23]: are we saying we don't need email to request content ?

File: src/pages/api/updateFeedback.ts
Comment (harikakotipalli) [inline src/pages/api/updateFeedback.ts:56]: I don't think I know this functionality- when do we update feedback ?

File: src/components/request-content-modal/request-content.modal.tsx
Comment (mcolella14) [reply src/components/request-content-modal/request-content.modal.tsx:23]: That's correct, Amy had [a PR that validated input](https://github.com/mongodb/devcenter/pull/388), and it was decided by Rachelle that email should be optional.

File: src/pages/api/updateFeedback.ts
Comment (mcolella14) [reply src/pages/api/updateFeedback.ts:56]: So we have different modal stages that each submit feedback. First the user clicks on the stars to rate the articles, which sends a POST request to `/api/createFeedback`, which enters that rating in our DB. Then they are shown a modal screen with checkboxes for why they like/dislike the article. when they hit "Next" on that modal, this endpoint is called, which updates that rating that was inserted in the POST request. Then they are shown a modal screen with text boxes for additional comments, which does the same thing by updating the document.

Comment (harikakotipalli) [reply src/pages/api/updateFeedback.ts:56]: LGTM. Read to approve once the conflicts are resolved.

Comment (harikakotipalli) [reply src/pages/api/updateFeedback.ts:56]: I would test kanopy to kanopy calls in staging after merging the code to staging.

**Review by harikakotipalli — APPROVED** [review]

**Review by justinmckenzie — APPROVED** [review]

File: src/pages/api/updateFeedback.ts
Comment (mcolella14) [reply src/pages/api/updateFeedback.ts:56]: @harikakotipalli Just tested in a preview environment and it seems to be working well, but I will definitely check in staging as well.

**Pull request merged**
